install/kubernetes/cilium/templates/hubble-ui/_nginx.tpl

```diff
@@ -0,0 +1,37 @@
+{{- define "hubble-ui.nginx.conf" }}
+server {
+    listen       8081;
+    listen       [::]:8081;
+    server_name  localhost;
+    root /app;
+    index index.html;
+    client_max_body_size 1G;
+
+    location / {
+        proxy_set_header Host $host;
+        proxy_set_header X-Real-IP $remote_addr;
+
+        # CORS
+        add_header Access-Control-Allow-Methods "GET, POST, PUT, HEAD, DELETE, OPTIONS";
+        add_header Access-Control-Allow-Origin *;
+        add_header Access-Control-Max-Age 1728000;
+        add_header Access-Control-Expose-Headers content-length,grpc-status,grpc-message;
+        add_header Access-Control-Allow-Headers range,keep-alive,user-agent,cache-control,content-type,content-transfer-encoding,x-accept-content-transfer-encoding,x-accept-response-streaming,x-user-agent,x-grpc-web,grpc-timeout;
+        if ($request_method = OPTIONS) {
+            return 204;
+        }
+        # /CORS
+
+        location /api {
+            proxy_http_version 1.1;
+            proxy_pass_request_headers on;
+            proxy_hide_header Access-Control-Allow-Origin;
+            proxy_pass http://127.0.0.1:8090;
```

Comment:
is this configuration also perform prefix rewrite like what we have before with envoy ?

Reply:
We have appropriate change on UI backend itself, now it actually listens on `/api` path https://github.com/cilium/hubble-ui/blob/bfefef23928e159f8416b42612097a83ba0ae67f/backend/main.go#L59-L63

Reply:
so does it mean that with this change hubble ui image must be at least v0.9.0+ ? If it's the case, what do you think about adding such validation here https://github.com/cilium/cilium/blob/master/install/kubernetes/cilium/templates/validate.yaml ?

Reply:
Correct. How could I run this validation to check if wrote this right?

Reply:
it will be run automatically as part of helm install or helm template with supplied values.